scaling_performance/using_cpu_manager.adoc

```diff
@@ -0,0 +1,193 @@
+        cpu: 1
+        memory: "1G"
+  nodeSelector:
+    cpumanager: "true"
+----
+
+. Create the pod:
++
+----
+# oc create -f cpumanager.yaml
+----
+
+. Verify that the pod is scheduled to the node that you labeled:
++
+----
+# oc describe pod cpumanager
+Name:         cpumanager-4gdtn
+Namespace:    test
+Node:         perf-node.example.com/172.31.62.105
+...
+    Limits:
+      cpu:     1
+      memory:  1G
+    Requests:
+      cpu:        1
+      memory:     1G
+...
+QoS Class:       Guaranteed
+Node-Selectors:  cpumanager=true
+                 region=primary
+----
+
+. Verify that the `cgroups` are set up correctly. Get the PID of the pause process:
++
+----
+# systemd-cgls -l
+├─1 /usr/lib/systemd/systemd --system --deserialize 20
+├─kubepods.slice
+│ ├─kubepods-pod0ec1ab8b_e1c4_11e7_bb22_027b30990a24.slice
+│ │ ├─docker-b24e29bc4021064057f941dc5f3538595c317d294f2c8e448b5e61a29c026d1c.scope
+│ │ │ └─44216 /pause
+----
++
+Pods of QoS tier `Guaranteed` are placed within the `kubepods.slice`. Pods of other
+QoS tiers end up in child `cgroups` of `kubepods`.
++
+----
+# cd /sys/fs/cgroup/cpuset/kubepods.slice/kubepods-pod0ec1ab8b_e1c4_11e7_bb22_027b30990a24.slice/docker-b24e29bc4021064057f941dc5f3538595c317d294f2c8e448b5e61a29c026d1c.scope
+# for i in `ls cpuset.cpus tasks` ; do echo -n "$i "; cat $i ; done
+cpuset.cpus 2
+tasks 44216
+----
+
+. Check the task's allowed CPU list:
++
+----
+# grep ^Cpus_allowed_list /proc/44216/status
+Cpus_allowed_list:      2
+----
+
+. Verify that another pod (in this case, the pod in the `burstable` QoS tier) on the
+system cannot run on the core allocated for the `Guaranteed` pod:
++
+----
+# cat /sys/fs/cgroup/cpuset/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-podbe76ff22_dead_11e7_b99e_027b30990a24.slice/docker-da621bea7569704fc39f84385a179923309ab9d832f6360cccbff102e73f9557.scope/cpuset.cpus
+0-1,3
+----
++
+----
+# oc describe node perf-node.example.com
+...
+Capacity:
+ cpu:     4
+ memory:  16266720Ki
+ pods:    40
+Allocatable:
+ cpu:     3500m
+ memory:  16164320Ki
+ pods:    40
+---
+  Namespace                  Name                      CPU Requests  CPU Limits  Memory Requests  Memory Limits
+  ---------                  ----                      ------------  ----------  ---------------  -------------
+  test                        cpumanager-4gdtn          1 (28%)       1 (28%)     1G (6%)          1G (6%)
+  test                        cpumanager-hczts          1 (28%)       1 (28%)     1G (6%)          1G (6%)
+  test                        cpumanager-r9wrq          1 (28%)       1 (28%)     1G (6%)          1G (6%)
+...
+Allocated resources:
+  (Total limits may be over 100 percent, i.e., overcommitted.)
+  CPU Requests  CPU Limits  Memory Requests  Memory Limits
+  ------------  ----------  ---------------  -------------
+  3 (85%)       3 (85%)     5437500k (32%)   9250M (55%)
+----
++
+This VM has four CPU cores. You set `kube-reserved` to 500 millicores, meaning
```

Comment:
Didn't we set this value to 1000 earlier in https://github.com/openshift/openshift-docs/pull/6750/files#diff-ac5bd8640a40585a2f744a1fcdc79fb4R57

Reply:
Good catch.  Must be a copy paste error on my part.  Let's just change that to 500m.  I put a comment to ask @ahardin-rh to do that.